add note for cgroup v1 deprecation during kubelet upgrade (#53082)
kubernetes/website: content/en/docs/tasks/administer-cluster/kubeadm/upgrading-linux-nodes.md

### Upgrade kubelet and kubectl

{{< note >}}
On Linux nodes, the kubelet defaults to supporting only cgroups v2.
For Kubernetes {{< skew currentVersion >}} the `FailCgroupV1` kubelet configuration option is set to true by default.

> Refer to the [Kubernetes cgroup v1 deprecation documentation](/docs/concepts/architecture/cgroups/#deprecation-of-cgroup-v1) for more information.

Reviewer: ```suggestion
Please refer to the [Kubernetes cgroup v1 deprecation documentation](/docs/concepts/architecture/cgroups/#deprecation-of-cgroup-v1).
```

### Upgrade kubelet and kubectl

{{< note >}}
FailCgroupV1 is a kubelet configuration option that is set to 'true' by default since v1.35.
If you are using cgroups v1, kubelet will fail to start if the FailCgroupV1 is not set..
The cgroups v1 support is deprecated and will be removed in a future release.
If you are using cgroups v1, you should migrate to cgroups v2.
To explicitly enable cgroups v1 support for kubelet v1.35 or newer,

> you must set the kubelet configuration option 'FailCgroupV1' to 'false'. You must also explicitly skip this validation.

Reviewer 2: ```suggestion
you **must** set the kubelet configuration option `FailCgroupV1` to false. You must also explicitly skip this kubeadm validation.
```
